docker/config/local_dev.env

```diff
@@ -54,14 +54,11 @@ resource.elasticsearch.elasticsearch_urls=http://elasticsearch:9200
 # boto
 # ----
 
-resource.boto.host=localstack-s3
-resource.boto.port=5000
+resource.boto.endpoint_url=http://localstack-s3:5000/
 resource.boto.access_key=foo
 secrets.boto.secret_access_key=foo
 resource.boto.bucket_name=dev_bucket
 resource.boto.temporary_file_system_storage_path=/tmp
-resource.boto.calling_format=boto.s3.connection.OrdinaryCallingFormat
```

Comment:
We had to switch calling format before because we had periods in the bucket names. That's no longer true, so we can use the defaults (Antenna does) and we don't have to fiddle with calling formats anymore.